### DISCUSSION TITLE
Has Droid-ify really rebranded to Neo-Store? Or are those two apps?

### DISCUSSION THREAD

**Discussion by TuringTux** [PrivSec-dev/privsec.dev#190] [Q&A] [answered]
I've just read https://privsec.dev/posts/android/f-droid-security-issues/ and am deciding to switch to a different F-Droid client to avoid some of the issues mentioned there (e.g. targeting an higher Android API level to make use of better sandbox features, not requiring a privileged extension with questionable security practices for batch updates while still allowing batch updates).

The post mentions that

> [Droid-ify](https://github.com/Iamlooker/Droid-ify) (recently rebranded to Neo-Store) seems to be a better option than the official client in most aspects.

However, both Droid-ify and Neo-Store seem to be alive and well:

* https://github.com/Droid-ify/client
* https://github.com/NeoApplications/Neo-Store

What has happened here? Should I use Droid-ify or Neo-Store instead of F-Droid, or are they equivalent from a security standpoint?

**Comment by wj25czxj47bu6q** [comment, marked as answer]
Parts of that article need to be updated (#151).

The TL;DR about Droid&#8209;ify and Neo&nbsp;Store is that Droid&#8209;ify rebranded to Neo&nbsp;Store, and then the original dev of Droid&#8209;ify forked the project to create the Droid&#8209;ify we currently have.

The best F&#8209;Droid client to use currently is actually **[F&#8209;Droid Basic](https://cloudflare.f-droid.org/en/packages/org.fdroid.basic/)** which is significantly less buggy than Neo&nbsp;Store and Droid&#8209;ify alike, supports repository mirrors properly, and removes an extra party of trust.

Just note that, if you'd like to disable the F&#8209;Droid main repository, you will need an alternative method to receive updates for F&#8209;Droid Basic, such as [Obtainium](https://github.com/ImranR98/Obtainium).

**Comment by TuringTux** [comment]
Thanks for the quick and helpful reply, that is good to know! If I can do something to help updating the article, let me know – apart from that, this issue can be considered solved, at least from my side.
